source/lmp/pair_deepmd.cpp

```diff
@@ -450,9 +450,7 @@ void PairDeepMD::compute(int eflag, int vflag) {
   if (numb_models == 0) {
     return;
   }
-  if (eflag || vflag) {
-    ev_setup(eflag, vflag);
-  }
+  ev_init(eflag, vflag);
```

Comment:
Consider adding a comment explaining why ev_init is used instead of ev_setup. This will help future developers understand the reasoning behind this change. [medium]